tests/functional/object/mcg/test_s3_vector.py

```
@@ -997,15 +998,36 @@ def test_vector_query_operations(self, vector_bucket_factory, mcg_obj):
+                5,
+                vector_utils.query_vectors,
                 s3vectors_client,
                 query_vector=query_vector,
                 top_k=top_k,
                 vectorBucketName=vector_bucket_name,
                 indexName=index_name,
                 returnDistance=True,
                 returnMetadata=True,
-            ).get("vectors", [])
+            ):
+                results_step6 = results_step6.get("vectors", [])
```

Ruff (0.15.18)

[warning] `for` loop variable `results_step6` overwritten by assignment target

(PLW2901)